Corona-Warn-App/src/main/res/values-ro/strings.xml

```diff
@@ -632,8 +632,10 @@
     <string name="settings_background_priority_headline">"Actualizare automată a stării de risc"</string>
     <!-- YTXT: settings(background priority) - description text -->
     <string name="settings_background_priority_body">"Dacă activați activitatea în fundal cu prioritate, aplicația poate determina în orice moment starea riscului dvs. Această opțiune dezactivează optimizarea vieții bateriei doar pentru aplicația Corona-Warn."</string>
-    <!-- XACT: settings(background priority) - illustraction description -->
```

Comment:
Strings should only be  added for DE and Default to avoid merge conflicts with the translation delivery.